ideascube/conf/idb_grc_malakasa.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
```

Comment:
(I know, I know, it's everywhere else but…) we can get read of this now that we are in python 3 :)